packages/nextra-theme-blog/package.json

```diff
@@ -28,7 +28,7 @@
     "dev:tailwind": "cross-env TAILWIND_MODE=watch pnpm postcss src/styles.css -o style.css --watch",
     "prepublishOnly": "pnpm build",
     "clean": "rimraf ./dist ./style.css",
-    "format": "prettier --write ."
+    "format": "prettier --ignore-path ../../.gitignore --write --list-different ."
```

Comment:
to ignore all files from `gitignore`, otherwise prettier prettify files from `dist/`
![image](https://user-images.githubusercontent.com/7361780/181854707-f5c249f7-8cb8-40d0-b323-8fbecdddbc78.png)